### PR TITLE
New Taskrouter Stats endpoints and responses

### PR DESCRIPTION
**Pull request by jwitz10**

### CODE REVIEW

File: rest/taskrouter/statistics/workflow/cumulative/example-1.6.x.py [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+# Your Account Sid and Auth Token from twilio.com/user/account
+account_sid = "ACXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX"
+auth_token = "your_auth_token"
+workspace_sid = "WSXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX"
+workflow_sid = "WWXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX"
+
+client = Client(account_sid, auth_token)
+
+statistics = client.taskrouter.workspaces(workspace_sid) \
+    .workflows(workflow_sid).cumulativestatistics().fetch()
```
Comment (acamino) [inline rest/taskrouter/statistics/workflow/cumulative/example-1.6.x.py]: @jwitz10, you may want to use `workers_cumulative_statistics`. https://github.com/twilio/twilio-python/blob/df0b500811ff9567d27aaa6c2099c360e2506f46/twilio/rest/taskrouter/v1/workspace/workflow/__init__.py#L386.

I found this method name quite verbose, what do you think about asking the Yoyodine team for something closer to what you have: `cumulative_statistics`?

File: rest/taskrouter/statistics/workflow/realtime/example-1.6.x.py [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+# Your Account Sid and Auth Token from twilio.com/user/account
+account_sid = "ACXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX"
+auth_token = "your_auth_token"
+workspace_sid = "WSXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX"
+workflow_sid = "WWXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX"
+
+client = Client(account_sid, auth_token)
+
+statistics = client.taskrouter.workspaces(workspace_sid) \
+    .workflows(workflow_sid).realtimestatistics().fetch()
```
Comment (acamino) [inline rest/taskrouter/statistics/workflow/realtime/example-1.6.x.py]: Same here, it should be `workflow_real_time_statistics`.

File: rest/taskrouter/statistics/workspace/cumulative/example-1.6.x.py [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+
+# Your Account Sid and Auth Token from twilio.com/user/account
+account_sid = "ACXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX"
+auth_token = "your_auth_token"
+workspace_sid = "WSXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX"
+
+client = Client(account_sid, auth_token)
+
+workspace = client.taskrouter.workspaces(workspace_sid)
+statistics = workspace.cumulativestatistics().fetch()
```
Comment (acamino) [inline rest/taskrouter/statistics/workspace/cumulative/example-1.6.x.py]: Same applies here: `workspace_cumulative_statistics`.

File: rest/taskrouter/statistics/workspace/realtime/example-1.6.x.py [hunk 4]
```diff
@@ -0,0 +1,15 @@
...
+
+# Your Account Sid and Auth Token from twilio.com/user/account
+account_sid = "ACXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX"
+auth_token = "your_auth_token"
+workspace_sid = "WSXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX"
+
+client = Client(account_sid, auth_token)
+
+workspace = client.taskrouter.workspaces(workspace_sid)
+statistics = workspace.realtimestatistics().fetch()
```
Comment (acamino) [inline rest/taskrouter/statistics/workspace/realtime/example-1.6.x.py]: And here `workspace_real_time_statistics`.

File: rest/taskrouter/statistics/workflow/cumulative/example-1.6.x.py (hunk 1 above)
Comment (jwitz10) [reply rest/taskrouter/statistics/workflow/cumulative/example-1.6.x.py]: Agreed.

**Comment by atbaker** [comment]
@jwitz10 Looks like we've still got some issues here: https://travis-ci.org/TwilioDevEd/api-snippets/builds/282950366?utm_source=github_status&utm_medium=notification

The Java 6 build is passing but it looks like there's some error with the Java 7 code.

I also spot checked the Python samples - it looks like the methods are being exposed as `workspace_cumulative_statistics` instead of just `cumulative_statistics`:

https://github.com/twilio/twilio-python/blob/5eb38fc352ad3dff935ec10fcaf1652fea466ca0/twilio/rest/taskrouter/v1/workspace/__init__.py#L482

**Comment by joliveros** [comment]
@jwitz10 @atbaker I forgot to mention I've got most of these snippets fixed now. I'll probably have them all fix by tomorrow morning at the latest.

**Comment by atbaker** [comment]
Awesome. Thank you @joliveros

**Pull request merged**